### PR TITLE
NAS-129442 / 24.10 / Remove unused packages

### PR DESCRIPTION
**Pull request by themylogin**

### CODE REVIEW

**Comment by themylogin** [comment]
time 2:00

File: requirements.txt [hunk 1]
```diff
@@ -16,21 +16,17 @@ certbot-dns-ovh==2.6.0
...
 jsonlines==3.1.0
 pyRFC3339==1.1
 git+https://github.com/truenas/python-prompt-toolkit.git@84d42cd#egg=prompt-toolkit==3.0.37
 protobuf==4.24.4
 pyroute2==0.7.9
 python-digitalocean==1.17.0
-PyYAML==6.0.1
 remote-pdb==2.1.0
 s3transfer==0.6.1
-tableprint==0.9.1
```
Comment (creatorcary) [inline requirements.txt:35]: We may actually need this for midcli.

**Review by creatorcary — CHANGES_REQUESTED** [review]
The log file I sent @themylogin suggests that tableprint is a necessary dependency.

File: requirements.txt (hunk 1 above)
Comment (themylogin) [reply requirements.txt:35]: Actually, we don't, midcli control file is at fault https://github.com/truenas/midcli/pull/109

**Review by creatorcary — APPROVED** [review]

**Pull request merged**